lumen/base.py

```diff
@@ -16,6 +18,8 @@ class Component(param.Parameterized):
 
     def __init__(self, **params):
         self._refs = params.pop('refs', {})
+        expected = list(self.param.params())
+        validate_parameters(params, expected, type(self).name)
```

Comment:
Reasonable at this level, note though that a lot of components simply collect any parameters that haven't been explicitly declared into `self.kwargs` so this will never be triggered in those cases.